api/credentials-config-service.yaml

```diff
@@ -216,6 +216,19 @@ components:
           $ref: '#/components/schemas/HolderVerification'
       required:
         - type
+    TrustedParticipantsList:
```

Comment:
I find the name a bit confusing tbh but I wouldn't add a "Entry" to the name. What would you think about something like TrustedParticipantEndpoint?

Reply:
TrustedParticipantsListEndpoint? I think list is important, because it contains the list of participants, not just a single one